### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [shubhamchaudhary/wordpowermadeeasy#10]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Word Power Made Easy](https://play.google.com/store/apps/details?id=com.wWordPowerMadeEasy.original&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Word Power Made Easy app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_175551](https://user-images.githubusercontent.com/25429624/99563167-494ed300-29c9-11eb-8849-da08b3301571.png)
